### PR TITLE
Add configuration to allow N8N to connect to the database

### LINKED ISSUE #924: Update postgres configuration to all connection from new N8N server
## Description
Currently N8N has database access, but it hasn't been set up for the new N8N server. It means some workflow won't work till we fix it 

## Expected Behavior
Workflow can use the database

## Actual Behaviour
Workflow can't use the database, and get an error like  :
```
no pg_hba.conf entry for host "...", user "...", database "...", SSL on
```

## Workaround
Keep using the old N8N server till the issue is fixed .

## Steps to Reproduce

1. 
2.
3.
4.

### PR DESCRIPTION
**Pull request by rioug**
Fix #924

Update `ph_hba.conf` to allow the new N8N server to connect to postgres database

I added both IPv4 and IPv6 entry, I am not sure if it's strictly necessary but it doesn't hurt to cover everything

### CODE REVIEW

**Review by mkllnk — COMMENTED** [review]
Great. Now those entries need to be added to the `custom_hba_entries` within the host variables, right?

File: inventory/host_vars/openfoodnetwork.org.au/config.yml [hunk 1]
```diff
@@ -28,8 +28,8 @@ postgres_listen_addresses:
 custom_hba_entries:
   - "{{ custom_hba_metabase }}"
   - "{{ custom_hba_n8n }}"
-  - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
-  - { type: hostssl, database: "{{ db }}", user: zapier, address: '167.99.89.242/32', auth_method: md5 } # for metabase (data.openfoodnetwork.org.au)
```
Comment (mkllnk) [inline inventory/host_vars/openfoodnetwork.org.au/config.yml:32]: I think that we still need these. Definitely Metabase. Zapier is also in use but I don't know if we use the database connection.

**Review by mkllnk — CHANGES_REQUESTED** [review]

File: inventory/host_vars/openfoodnetwork.org.au/config.yml (hunk 1 above)
Comment (rioug) [reply inventory/host_vars/openfoodnetwork.org.au/config.yml]: I re enable Zappier for those that weren't commented out. Sorry I was under the impression we moved to N8N and were not using Zappier any more.

**Review by mkllnk — APPROVED** [review]
Great.

I also thought that we moved away from Zapier but I saw some integration posting on Slack the other day. So something is still running there.

**Comment by RachL** [comment]
@mkllnk Yes I can confirm FR did not migrate entirely on n8n and Zapier is still in use.

**Comment by dacook** [comment]
Provisioned for all staging, and most prod servers. But four failed 😞 . Looks like the same issue for each:

```
TASK [oefenweb.swapfile : install dependencies] ************************************************************************
failed: [app.katuma.org] (item=['util-linux']) => {"ansible_loop_var": "item", "changed": false, "item": ["util-linux"], "msg": "Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., W:Skipping acquire of configured file 'nginx/binary-i386/Packages' as repository 'http://nginx.org/packages/ubuntu bionic InRelease' doesn't support architecture 'i386', E:The repository 'http://apt.postgresql.org/pub/repos/apt bionic-pgdg Release' no longer has a Release file."}

TASK [oefenweb.swapfile : install dependencies] ************************************************************************
failed: [csicsoka.org] (item=['util-linux']) => {"ansible_loop_var": "item", "changed": false, "item": ["util-linux"], "msg": "Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., E:The repository 'http://apt.postgresql.org/pub/repos/apt bionic-pgdg Release' no longer has a Release file."}

TASK [oefenweb.swapfile : install dependencies] ************************************************************************
failed: [openfoodnetwork.net] (item=['util-linux']) => {"ansible_loop_var": "item", "changed": false, "item": ["util-linux"], "msg": "Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., E:The repository 'http://apt.postgresql.org/pub/repos/apt bionic-pgdg Release' no longer has a Release file."}

TASK [oefenweb.swapfile : install dependencies] ************************************************************************
failed: [openfoodnetwork.org.au] (item=['util-linux']) => {"ansible_loop_var": "item", "changed": false, "item": ["util-linux"], "msg": "Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., E:The repository 'http://apt.postgresql.org/pub/repos/apt bionic-pgdg Release' no longer has a Release file."}
```

**Comment by dacook** [comment]
It’s probably not related to the PR, maybe our version of Postgres or Ubuntu is just too old....

But this error happened before the app config update step, so the config hasn't been updated yet. I think we should use `--start-at-task="template files"` to apply this config update for now.

**Comment by rioug** [comment]
This is due to postgres moving ubuntu 18:04 repo to apt-archive.postgresql.org, see : https://www.postgresql.org/message-id/ZN4OigxPJA236qlg%40msg.df7cb.de
This is not worth fixing at this stage, we are about to upgrade the servers.
I manually updated the 4 failing servers, all good now !

**Pull request merged**